packages/near-membrane-dom/src/browser-realm.ts

```diff
@@ -98,6 +99,7 @@ export default function createVirtualEnvironment(
     });
     linkIntrinsics(env, blueWindow, redWindow);
     linkUnforgeables(env, blueRefs, redRefs);
+    setupStackTrace(redWindow);
```

Comment:
Is there a particular order here? Why after `linkUnforgeables()` and before `tameDOM()`?

Reply:
No. I had originally included it in `linkIntrinsics`. I can move it before.